fin/src/include/conv_fin.hpp

```diff
@@ -691,7 +694,8 @@ int ConvFin<Tgpu, Tref>::MIOpenPerfEval()
                     std::cerr << solver_name << " Finished Search WRW" << std::endl;
                     kern_objs = BuildJsonKernelList(h, solution.construction_params);
                     SolutionHasProgram(h, solution);
-                    params = s.GetPerfCfgParams(ctx, db);
+                    // todo: proper handling of empty values
+                    params = s.GetPerfCfgParams(ctx, db).value_or("");
```

Comment:
@cderb I do not know how to properly handle empty `params` here. But what is clear is that your approval for fin changes is required.

Reply:
It is ok if params are set to empty string here.

Since Fin is a separate repository from MIOpen and will be setup as a submodule, it is preferable for these changes to be submitted to https://github.com/ROCmSoftwarePlatform/MIFin if they are necessary.